src/SharpBrick.PoweredUp/Devices/SimpleMediumLinearMotor.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using SharpBrick.PoweredUp.Protocol;
+using SharpBrick.PoweredUp.Utils;
+
+namespace SharpBrick.PoweredUp;
+
+/// <summary>
+/// SimpleMediumLinearMotor, advertised as Motor, only supports power mode, this works for the WoDo 2.0 Medium (LPF2-MMOTOR)
+/// </summary>
+public class SimpleMediumLinearMotor : BasicMotor, IPoweredUpDevice
+{
+    public SimpleMediumLinearMotor()
+        : base()
+    { }
+    public SimpleMediumLinearMotor(ILegoWirelessProtocol protocol, byte hubId, byte portId)
+        : base(protocol, hubId, portId)
+    { }
+
+    public IEnumerable<byte[]> GetStaticPortInfoMessages(Version softwareVersion, Version hardwareVersion, SystemType systemType)
+      // Dump taken from LEGO 6290182 - 21980 - Electric, Motor WeDo 2.0 Medium which reports as 'LPF2-MMOTOR'.
```

Comment:
IMHO, the motor is not reporting that but your `MediumTechnicHub` describes the motor. The Lego Wireless Protocol is a Bluetooth protocol, not a wire protocol between the hub and the motor. It could though, but I am not aware of any details there.

Reply:
I am not sure, but of course we communicate with the Hub, but I think the data essentially comes from the motor and is translated and passed on by the hub, don't all motors work with all hubs? All hubs would have to know all device types then.